plugins/metrics/tests/test_pipeline.py

```
@@ -0,0 +1,779 @@
+        }
+        payload = {
+            "resourceLogs": [{
+                "resource": {"attributes": [{"key": "service.name", "value": {"stringValue": "claude-code"}}]},
+                "scopeLogs": [{"logRecords": [safe_log, dirty_log]}],
+            }]
+        }
+        _post("/v1/logs", payload)
+        _flush(col)
+
+        logs = _logs_from_jsonl(col.jsonl_path)
+        bodies = [l["body"] for l in logs]
```

Ruff (0.15.17)

[error] Ambiguous variable name: `l`

(E741)